drone/oc-litmus/frontend.toml

```diff
@@ -0,0 +1,63 @@
+[shared]
+jwt_secret = "Pive-Fumkiu4"
+gatewaysvc = "localhost:19000"
+
+# This frontend.toml config file will start a reva service that:
+# - serves as the entrypoint for owncloud APIs.
+# - serves http endpoints on port 20080
+#   - / --------------- ocdav
+#   - /ocs ------------ ocs
+#   - /oauth2 --------- oidcprovider
+#   - /.well-known ---- wellknown service to announce openid-configuration
+#   - TODO(diocas): ocm
+# - authenticates requests using oidc bearer auth and basic auth as fallback
+# - serves the grpc services on port 18002
```

Comment:
```suggestion
# - serves the grpc services on port 20099
```
Do we even want this duplication of the port number in a comment? It just gets out-of-sync with the port number directly below.